.demo/env/charts/gatekeeper-operator.yaml

```diff
@@ -1,3 +1,22 @@
 charts:
     gatekeeper-operator:
         enabled: true
+        replicas: 1
+        disableValidatingWebhook: false
+        auditFromCache: false
+        emitAuditEvents: false
+        emitAdmissionEvents: false
+        constraintViolationsLimit: 20
+        auditInterval: 60
+        logLevel: INFO
+
+        excludedNamespaces:
+            - 'knative-serving'
+            - 'shared'
```

Comment:
Why do exclude shared namespace?

Reply:
I think that got mixed up from other older values. 
It can be safely removed.

`"Knative"' also released annotations support couple of days ago  🥇 -> [here](https://github.com/knative/operator/pull/472)